docker-compose/compose.yaml

```diff
@@ -170,22 +204,35 @@ services:
     ports:
       - "5601:5601"
     depends_on:
-      - elasticsearch
+      elasticsearch:
+        condition: service_healthy
+    healthcheck:
+      test: ["CMD-SHELL", "curl -f http://localhost:5601/status"]
+      interval: 10s
+      timeout: 10s
+      retries: 5
+      start_period: 30s
   search:
     image: quay.io/alfresco/alfresco-elasticsearch-live-indexing:4.1.0
     mem_limit: 1g
     depends_on:
       elasticsearch:
-        condition: service_started
-      search-reindexing:
-        condition: service_completed_successfully
+        condition: service_healthy
+      transform-core-aio:
+        condition: service_healthy
```

Comment:
also filestore?

I would try to be consistent also with the position of these declaration, let's decide if before or after the healthchecks